app/src/main/java/org/jellyfin/androidtv/preferences/UserPreferences.kt

```diff
@@ -43,6 +40,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(PreferenceManage
 	 */
 	var backdropEnabled by booleanPreference("pref_show_backdrop", true)
 
+	/**
+	 * Enable background images while browsing
+	 */
+	var gridDirection by enumPreference("grid_direction", GridDirection.VERTICAL)
```

Comment:
Should we default to horizontal since that is the current layout? I honestly don't know lol

Reply:
Vertical is basically the same as web so it makes it "more consistent". Also, I just noted the description for this property is wrong lol, can you find out where I copied the code from? 🤣

Reply:
I did change it to horizontal because the vertical one won't work that well until we rewrite the code, I also marked it as "experimental".